php_phongo.c

```diff
@@ -1742,22 +1856,22 @@ zend_object_iterator *php_phongo_cursor_get_iterator(zend_class_entry *ce, zval
 /* {{{ Memory allocation wrappers */
 static void* php_phongo_malloc(size_t num_bytes) /* {{{ */
 {
-	return emalloc(num_bytes);
+	return pemalloc(num_bytes, 1);
```

Comment:
Persistent memory allocation is required since clients are persisted between requests. This means that all BSON allocation will also use `malloc()` instead of `emalloc()`, even though much of that is only relevant inside a single request. Provided libmongoc and libbson don't have their own memory leaks, this shouldn't be an issue, though.